core/src/components/popover-menu/popover-menu-item.scss

```diff
@@ -29,7 +29,7 @@ ul.sdds-popover-menu-wrapper {
       display: flex;
       flex-wrap: nowrap;
 
-      i {
+      tds-icon {
```

Comment:
Im guessing we'll update the popover to use our component instead of the webfont. Nice

Reply:
In this story we had situation like this:
`<i>
   <tds-icon name="truck"></tds-icon>
</i>
`
so I dropped `<i> `and just left `<tds-icon>` and assigned styling rules directly to it. 
It seems that only in the button component we could choose between the web font and web component version of icon.